salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -131,6 +154,28 @@ Wait minion available:
 
 {%- endif %}
 
+{#- Nodes in 2.6 or lower rely on manual provisioning of loop devices, which
+    we need to clean-up to avoid having duplicates generated by the systemd
+    units, introduced in 2.7
+    Note that we run this before `metalk8s.roles.etcd`, since this would
+    provision the aforementioned loop devices (because `etcd` role includes
+    `node` role, which takes care of volume provisioning) #}
+{%- if cleanup_loop_devices %}
+
+Cleanup existing loop devices:
+  salt.state:
+    - tgt: {{ node_name }}
+    - saltenv: metalk8s-{{ version }}
+    - sls:
+      - metalk8s.volumes.cleanup
+    - require:
+      - salt: Wait minion available
+      - metalk8s_drain: Drain the node
+    - require_in:
+      - salt: Check pillar before etcd deployment
```

Comment:
Hmm it means this orchestrate will fail if we are not on an etcd node

Reply:
Added a check